boards.txt

```diff
@@ -115,6 +115,12 @@ rpipico.menu.usbstack.picosdk=Pico SDK
 rpipico.menu.usbstack.picosdk.build.usbstack_flags="-I{runtime.platform.path}/tools/libpico"
 rpipico.menu.usbstack.tinyusb=Adafruit TinyUSB
 rpipico.menu.usbstack.tinyusb.build.usbstack_flags=-DUSE_TINYUSB "-I{runtime.platform.path}/libraries/Adafruit_TinyUSB_Arduino/src/arduino"
+rpipico.menu.usbstack.picosdknoser=Pico SDK (No Serial)
+rpipico.menu.usbstack.picosdknoser.build.usbstack_flags=-DDISABLE_USB_SERIAL "-I{runtime.platform.path}/tools/libpico"
+rpipico.menu.usbstack.tinyusbnoser=Adafruit TinyUSB (No Serial)
+rpipico.menu.usbstack.tinyusbnoser.build.usbstack_flags=-DDISABLE_USB_SERIAL -DUSE_TINYUSB "-I{runtime.platform.path}/libraries/Adafruit_TinyUSB_Arduino/src/arduino"
```

Comment:
Maybe I miss it, why would you want to have No Serial option ? Anyway `DISABLE_USB_SERIAL` does not work with TinyUSB though, however Serial CDC can be omitted by just simply called `Serial.end()` when using TinyUSB like this example https://github.com/adafruit/Adafruit_TinyUSB_Arduino/blob/master/examples/CDC/no_serial/no_serial.ino#L26

Therefore I would suggest to just drop No Serial option for TinyUSB.

Reply:
At first I wanted to have the No USB option, then I added No Serial just for the sake of completeness.

I will follow your suggestion and remove No Serial for TinyUSB.